packages/app/components/token-picker/TokenPicker.tsx

```diff
@@ -92,17 +160,58 @@ const TokenListRow = ({ closeAction, token }: TokenListRowProps) => (
   </div>
 );
 
-const TokenList = ({ closeAction, tokenList }: TokenListProps) => (
-  <div
-    className="mt-5 overflow-y-auto h-72 border-t border-surface-50 divide-y divide-surface-50"
-    id="tokenPickerList"
-  >
-    {tokenList.map(token => (
-      <TokenListRow
-        closeAction={closeAction}
-        key={token.address}
-        token={token}
-      />
-    ))}
-  </div>
-);
+const TokenList = ({
+  closeAction,
+  onClearSearchClick,
```

Comment:
[On the same note ](https://github.com/SwaprHQ/stackly-ui/pull/29/files#r1262341130)

I would remove click from the name.

```suggestion
  onClearSearch,
```